coalib/bearlib/languages/definitions/g4.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class g4:
+    aliases = 'antlr4', 'grun'
```

Comment:
I think `grun` is an alias for their test benching code, probably doesn't fit in here.

Reply:
Yeah but don't you think that ```grun ``` is also a part of ```antlr-4.7.1-complete.jar``` i.e both of these aliases ```antr4``` and ```grun``` is part of antlr.jar because antr4 testrig is used for testing grammar.

Reply:
Yeah, so the point of aliases is not to focus on the features of ANTLR's jar, the point of aliases in this file is to focus on language specific features of the grammar files, of which grun is not a part of

Also please not that `grun` is not any officially endorsed thing, it's just a bash shell alias (similar to `ll` which is given to `org.antlr.v4.gui.TestRig` for convenience sake, it could very well be any arbitrary name which we want.

Reply:
yes, even same case is applied to ```antlr4``` to which is also an alias for bash shell.i will do the changes also i will change language name as ```antlr``` and put on versions.Then it will be ```aliases = 'antlr4', 'antlr3' ``` just check gitter chat.